src/main/java/Duke.java

```diff
@@ -93,7 +93,7 @@ public static void insertPreviousItems(File f, ArrayList<Task> tasks) {
 
     public static void updateFile(ArrayList<Task> tasks) { 
         try {
-            File f = new File("data/stored.txt");
+            File f = new File("../data/stored.txt");
```

Comment:
It looks like this Path is used twice (line 28 as well), so could it be better to make it a constant?

https://nus-cs2113-ay2223s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers